labelling_web_app/scripts/update_local_labels.py

```diff
@@ -42,10 +43,15 @@ def main(args):
     for image_id, boxes in labels.items():
         if not os.path.isfile(os.path.join(labels_dir, '{}.txt'.format(image_id))):
             print('Downloading new image {}'.format(image_id))
-            urlretrieve(b2.get_download_url(image_id), os.path.join(images_dir, '{}.png'.format(image_id)))
-            label_file = os.path.join(labels_dir, '{}.txt'.format(image_id))
-            with open(label_file, 'w') as f:
-                f.write('\n'.join(['0 {}'.format(' '.join(map(str, x))) for x in boxes]))
+            try:
+                urlretrieve(b2.get_download_url(image_id), os.path.join(images_dir, '{}.png'.format(image_id)))
```

Comment:
Para verificar por 404: https://stackoverflow.com/questions/1308542/how-to-catch-404-error-in-urllib-urlretrieve